packages/@lightningjs/ui-components/src/components/ListItem/ListItem.js

```diff
@@ -111,7 +111,17 @@ export default class ListItem extends Button {
   }
 
   _updateTruncation() {
-    super._updateTruncation();
+    if (this._Title) {
```

Comment:
Does this work if moved to _updateTitle? Just wondering if it makes more sense to live in that method if possible.

Reply:
Tested it out and it doesn't work if I move it out of here.